public/manifest.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "SendToKodi",
   "short_name": "SendToKodi",
-  "options_page": "options.html",
+  "options_ui": {
+     "page": "options.html"
+  },
```

Comment:
This may not be strictly necessary but resolves a warning in Firefox because apparently `options_page` is [deprecated](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_page). Still need to confirm this doesn't break anything in Chrome.

Reply:
Checked, doesn't break anything in Chrome. Also I missed this before but it's not just the warning in Firefox, `options_page` is ignored so the only way to get to the options page is when it opens automatically on the first run.